app/assets/stylesheets/modules/_course.styl

```diff
@@ -205,3 +205,9 @@ rating(color)
 
     .details-form
       display block
+
+.multi-wiki [class$='multiValue']:first-of-type svg
```

Comment:
Nice solution. I guess react-select doesn't support per-option configuration of whether to show the delection UI?

Reply:
Can you get rid of the hover styling as well for this?

Also, I think it would be good to hide the dropdown for the home_wiki until the user begins typing, since we don't show any options in the dropdown before then.

![no svg but still hover](https://user-images.githubusercontent.com/848483/60359234-4b0b9700-998d-11e9-92ce-b543263b1aba.png)